Earthfile

```diff
@@ -436,21 +444,33 @@ netboot:
 
 arm-image:
   ARG OSBUILDER_IMAGE
+  ARG COMPRESS_IMG=true
   FROM $OSBUILDER_IMAGE
   ARG MODEL=rpi64
   ARG IMAGE_NAME=${FLAVOR}.img
   WORKDIR /build
-  ENV STATE_SIZE="6200"
-  ENV RECOVERY_SIZE="4200"
+  # These sizes are in MB
   ENV SIZE="15200"
-  ENV DEFAULT_ACTIVE_SIZE="2000"
+  IF [[ "$FLAVOR" =~ ^ubuntu* ]]
```

Comment:
I hate this. This should probably come from some env files where its clear what goes with what flavor and such. Not for this PR

Reply:
Good point, can it be done in the `/etc/elemental/config.yaml` that we include?

Reply:
ah probably not cuz this here has precedence when creating the image ... ok will open an issue